src/test/java/homework15/Base.java

```diff
@@ -0,0 +1,53 @@
+package homework15;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+
+import java.time.Duration;
+
+public class Base {
+
+    static WebDriver driver;
+    static WebDriverWait wait;
+
+    @BeforeTest
+    public static void beforeTest() {
+        System.out.println("------------ Testing started ----------------");
+    }
+
+    @AfterTest
+    public void afterTest() {
+        System.out.println("------------ Testing finished ----------------");
+    }
+
+
+    @BeforeMethod
+    public void beforeMethod() {
+        WebDriverManager.chromedriver().setup();
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--no-sandbox");
+        driver = new ChromeDriver(options);
+        driver.manage().window().setSize(new Dimension(1920, 1080));
+        driver.get("https://www.bstackdemo.com/");
+        wait = new WebDriverWait(driver, Duration.ofSeconds(2));
```

Comment:
No need to use it in Before method as explicit waits should be applied directly to elements